hello-thirdparty/gpg-sdk/build.gradle

```diff
@@ -0,0 +1,54 @@
+/*
+    sub module to download the Google Play games
+ */
+apply plugin: 'java'
+project.ext {
+    gpg_sdk_link = 'https://developers.google.com/games/services/downloads/gpg-cpp-sdk.v1.4.1.zip'
+}
+task fetch_gpg_sdk(dependsOn:'unzip_gpg_sdk') {
```

Comment:
maybe remove `fetch` and just use `unzip` as the top level.

I was expecting `fetch` to be the thing that fetch the sdk over http.